src/openrct2/Context.cpp

```diff
@@ -654,37 +654,37 @@ namespace OpenRCT2
                 bool sendMap = false;
                 if (info.Type == FILE_TYPE::SAVED_GAME)
                 {
-                    if (network_get_mode() == NETWORK_MODE_CLIENT)
+                    if (_network.GetMode() == NETWORK_MODE_CLIENT)
                     {
-                        network_close();
+                        _network.Close();
                     }
                     game_load_init();
-                    if (network_get_mode() == NETWORK_MODE_SERVER)
+                    if (_network.GetMode() == NETWORK_MODE_SERVER)
                     {
                         sendMap = true;
                     }
                 }
                 else
                 {
                     scenario_begin();
-                    if (network_get_mode() == NETWORK_MODE_SERVER)
+                    if (_network.GetMode() == NETWORK_MODE_SERVER)
                     {
                         sendMap = true;
                     }
-                    if (network_get_mode() == NETWORK_MODE_CLIENT)
+                    if (_network.GetMode() == NETWORK_MODE_CLIENT)
                     {
-                        network_close();
+                        _network.Close();
                     }
                 }
                 // This ensures that the newly loaded save reflects the user's
                 // 'show real names of guests' option, now that it's a global setting
                 peep_update_names(gConfigGeneral.show_real_names_of_guests);
                 if (sendMap)
                 {
-                    network_send_map();
+                    _network.Server_Send_MAP();
```

Comment:
PS: Something's off with whatever the naming standard was for this `Server_Send_MAP` :P